collab/foraging/toolkit/subsampling.py

```diff
@@ -20,14 +20,21 @@ def subset_frames_evenly_spaced(df_raw, desired_frames=300):
     return df
 
 
-def rescale_to_grid(df, size):
-    def rescale_column(column, size=size):
-        mapped = (column - column.min()) / (column.max() - column.min())
+# updated function to allow user to pass min and max
+def rescale_to_grid(df, size, gridMin=None, gridMax=None):
+    def rescale_column(column, size=size, gridMin=None, gridMax=None):
```

Comment:
As is, gridMin and gridMax of rescale_to_grid are not used at all, as you reset them in rescale_column.

Reply:
Talked about this in a meeting, no changes needed - ready to merge!